scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -227,7 +228,7 @@ public static BuildTargetResult PublishCoreHostPackages(BuildTargetContext c)
         {
             foreach (var file in Directory.GetFiles(Dirs.CorehostLocalPackages, "*.nupkg"))
             {
-                var hostBlob = $"{Channel}/Binaries/{CliNuGetVersion}/{Path.GetFileName(file)}";
+                var hostBlob = $"{Channel}/Binaries/{SharedFrameworkNugetVersion}/{Path.GetFileName(file)}";
```

Comment:
SharedHostNuGetVersion is defined as:

```
SharedHostNugetVersion = c.BuildContext.Get<HostVersion>("HostVersion").LockedHostVersion;
```

Where LockedHostVersion is:

```
        public string LockedHostVersion => IsLocked ? "1.0.1-rc2-002468-00" : LatestHostVersion;
```

So, if IsLocked is turned on, it will use a hard-coded version, and not the current version.